Create OOP-L1.cpp

eorit opened the pull request:
Lesson1

eorit committed:
Create OOP-L1.cpp

Ivan-Igorevich commented on OOP-L1.cpp:
```diff
@@ -0,0 +1,203 @@
+};
+
+void Stack::reset() {
+    quantity = -1;
```
.. и очищает значения

Ivan-Igorevich commented on OOP-L1.cpp:
```diff
@@ -0,0 +1,203 @@
+class Stack {
+private:
+    
+    int stack[10] = { 0 };
```
от констант бы избавиться, по хорошему

Ivan-Igorevich commented on OOP-L1.cpp:
```diff
@@ -0,0 +1,203 @@
+        return temp;
+    }
+    else if (quantity == -1) {
+        std::cout << "Stack is empty" << std::endl;
```
и ничего не вернули? bad style

eorit committed:
Update OOP-L1.cpp

correction of comments:
92 added a variable STACK_LENGTH
96 added reset value of stack in function "reset"
134 added return if stack is empty in function "pop"
